apps/omg_api/mix.exs

```diff
@@ -25,7 +25,7 @@ defmodule OMG.API.MixProject do
         ethereum_status_check_interval_ms: 6_000,
         child_block_minimal_enqueue_gap: 1
       ],
-      extra_applications: [:sentry, :logger],
+      extra_applications: [:sentry, :logger, :appsignal],
```

Comment:
`extra_applications` shouldn't need to be extended, if `:appsignal` is in `deps`. Please double check, but all should work fine without these, cf. [here](https://www.amberbit.com/blog/2017/9/22/elixir-applications-vs-extra_applications-guide/)